### PR TITLE
Swap latest callouts for fka versionless inline

### PR DESCRIPTION
**Pull request by joellabes**
## What are you changing in this pull request and why?
[As discussed in Slack](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1733449411561619?thread_ts=1733149843.050859&cid=C02NCQ9483C) - swap callouts for a more inline explanation of the changes, for blog posts where Versionless isn't the main focus so the rename doesn't matter as much. 

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

### CODE REVIEW

File: website/blog/2024-04-22-extended-attributes.md [hunk 1]
```diff
@@ -84,7 +80,7 @@ All you need to do is configure an environment as staging and enable the **Defer
 
 ## Upgrading on a curve
 
-Lastly, let’s consider a more specialized use case. Imagine we have a "tiger team" (consisting of a lone analytics engineer named Dave) tasked with upgrading from dbt version 1.6 to the new **Latest release track**, to take advantage of new features and performance improvements. We want to keep the rest of the data team being productive in dbt 1.6 for the time being, while enabling Dave to upgrade and do his work with Latest (and greatest) dbt.
+Lastly, let’s consider a more specialized use case. Imagine we have a "tiger team" (consisting of a lone analytics engineer named Dave) tasked with upgrading from dbt version 1.6 to the new **[Latest release track](/docs/dbt-versions/cloud-release-tracks)**, to take advantage of new features and performance improvements. We want to keep the rest of the data team being productive in dbt 1.6 for the time being, while enabling Dave to upgrade and do his work with Latest (and greatest) dbt.
```
Comment (joellabes) [inline website/blog/2024-04-22-extended-attributes.md:83]: This article was written during the Keep on Latest Version phase, so it never talked about versionless and explaining that we used to call it versionless was going to mess with the screenshots! So I've just linked to release tracks.

File: website/blog/2024-06-12-putting-your-dag-on-the-internet.md [hunk 2]
```diff
@@ -112,12 +108,12 @@ The result is a model that will keep track of dbt invocations, and the current U
...
 
-**A few words about the power of Commercial Open Source Software**
+## A few words about the power of Commercial Open Source Software
 
 In order to get this functionality shipped quickly, EQT opened a pull request, Snowflake helped with some problems we had with CI and a member of dbt Labs helped write the tests and merge the code in!  
 
-dbt now features this functionality in dbt 1.8+ and the "Latest" release track in dbt Cloud (dbt overview [here](/docs/dbt-versions/cloud-release-tracks)). 
+dbt now features this functionality in dbt 1.8+ and all [Release tracks](/docs/dbt-versions/cloud-release-tracks) in dbt Cloud.
```
Comment (joellabes) [inline website/blog/2024-06-12-putting-your-dag-on-the-internet.md:117]: Likewise, this feature is old enough that it's in every release track, not just the Latest one, so saying Latest fka Versionless was less accurate.

File: website/blog/2024-06-12-putting-your-dag-on-the-internet.md [hunk 3]
```diff
@@ -49,7 +45,7 @@ This API is open and if it requires an API key, handle it similarly to managing
 For simplicity’s sake, we will show how to create them using [pre-hooks](/reference/resource-configs/pre-hook-post-hook) in a model configuration yml file:
 
 
-```
+```yml
```
Comment (joellabes) [inline website/blog/2024-06-12-putting-your-dag-on-the-internet.md:48]: Fixed a couple of markdownlint warnings while I was here

**Review by runleonarun — APPROVED** [review]
These changes look good, Joel! Thanks for your keen eye 👁️!

**Pull request merged**